mailbox/event/event-rabbitmq/src/test/java/org/apache/james/mailbox/events/RabbitMQEventBusTest.java

```diff
@@ -326,17 +323,25 @@ void startShouldCreateEventExchange() {
             }
 
             @Test
-            void dispatchShouldWorkAfterNetworkIssuesForOldRegistration() throws Exception {
+            void dispatchShouldWorkWhileNetworkIssuesForOldRegistration() throws Exception {
```

Comment:
How is that possible? Does it mean that 'pause' is not enough to simulate well a network issue?

Reply:
Before, this test come true because we create new channel during `pause`.
As this test I did, `pause` is not enough to simulate a network issue.

Reply:
True. This is closer to the "GC pause".

If the pause is not long enough then the session / connection is never broken hence this strange behaviour.

We might want to see what happens when combining it with a sleep - at some points I suspect that it will fail (60s + ?)

Reply:
Combining with sleep is still not simulate network issue. What new idea for that.